tasks/tiler/xyz.py

```diff
@@ -73,12 +80,23 @@ def _get_config_data(self):
             return json.load(f)
 
     def _get_table_schema(self, table_config):
+        mc_dates = [date.replace('-', '') for date in self._get_mc_dates()]
         columns = {}
-        for dataset, col_data in table_config['columns'].items():
-            columns[dataset] = []
-            for column in col_data:
-                nullable = '' if column['nullable'] else 'NOT NULL'
-                columns[dataset].append("{} {} {}".format(column['column_name'], column['type'], nullable))
+
+        columns['do'] = []
+        for do_column in table_config['columns']['do']:
+            nullable = '' if do_column['nullable'] else 'NOT NULL'
+            columns['do'].append("{} {} {}".format(do_column['column_name'], do_column['type'], nullable))
+
+        columns['mc'] = []
+        for mc_date in mc_dates:
```

Comment:
Should not be this month instead of date? We are creating a column by date instead of by month

Reply:
Yes, they need a column by each date. The input is the month and for that month we look for every date with that month and create a column with each date among those

Reply:
So how many columns?

Reply:
Depends on the dates that we have in our mc data for a given month. Currently, 5 columns for each month, totalling `number_of_mccolumns x n_categories(currently 5) x n_dates(currently 5)`

Reply:
Ok, this could lead to a ton of columns xD

Reply:
yes, that's what we talked about before... splitting by months, splitting in several tables, adding all the dates into an array. If I recall correctly we decided to do it this way for this iteration and then think about the improvements if needed, isn't it?

Reply:
Yep, it's just I thought it was by month instead of by date hehe